tests/test_modeling_common.py

```diff
@@ -3753,177 +3753,194 @@ def get_mean_reldiff(failcase, x, ref, atol, rtol):
                 if not has_sdpa and model_sdpa.config.model_type != "falcon":
                     raise ValueError("The SDPA model should have SDPA attention layers")
 
-                # We use these for loops instead of parameterized.expand just for the interest of avoiding loading/saving 8 times the model,
+                # We use these for loops instead of parameterized.expand just for the interest of avoiding loading/saving 16 times the model,
                 # but it would be nicer to have an efficient way to use parameterized.expand
                 fail_cases = []
                 for padding_side in ["left", "right"]:
                     for use_mask in [False, True]:
-                        for batch_size in [1, 5]:
-                            dummy_input = inputs_dict[model.main_input_name]
-
-                            if dummy_input.dtype in [torch.float32, torch.bfloat16, torch.float16]:
-                                dummy_input = dummy_input.to(torch_dtype)
+                        for output_attentions in [True, False]:
+                            if (
+                                not (
+                                    self.has_attentions
+                                    and "output_attentions" in inspect.signature(model_sdpa.forward).parameters
+                                )
+                                and output_attentions
+                            ):
```

Comment:
can we split this in 2 lines? 
```suggestion
                            can_output_attn = "output_attentions" in inspect.signature(model_sdpa.forward).parameters
                            if (not (self.has_attentions and can_output_attn) and output_attentions):
```